### PR TITLE
(Docs): Added 24pullRequests announcement post

### PR DESCRIPTION
**Pull request by Abhijay007**
# Summary of Changes

Added 24pullRequests announcement post

## Checklist

- [x] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
- [x] My code is well-documented, and I've updated relevant documentation.
- [x] I have tested these changes on my local environment.
- [x] All tests pass, and there are no new linting errors.
- [x] I have reviewed and proofread my code and the changes.
- [x] The branch is up-to-date with the base branch.

### CODE REVIEW

File: blog/2023-12-01-24pullRequests/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Uptane Joins 24 Pull Requests for a Season of Open Source Giving ❄️"
+authors:
+  - Abhij
+---
+
+![](bannerImage.png)
+As the winter season approaches, [Uptane](https://uptane.org/) is thrilled to announce its participation in the annual 24 Pull Requests event! This initiative is a testament to our commitment to collaboration and community building.
```
Comment (jhdalek55) [inline blog/2023-12-01-24pullRequests/index.md:8]:
```suggestion
As winter approaches, [Uptane](https://uptane.org/) is thrilled to announce its participation in the annual 24 Pull Requests event! This initiative is a testament to our  project's continuing commitment to collaboration and community building.

```

File: blog/2023-12-01-24pullRequests/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## 🤝 About Uptane:
+
+Uptane stands as a robust framework designed to fortify the security of software updates within the automotive industry. Uptane addresses the need for a reliable solution, ensuring the safety and integrity of software updates for vehicles.
```
Comment (jhdalek55) [inline blog/2023-12-01-24pullRequests/index.md:12]:
```suggestion
Uptane is a robust framework designed to fortify the security of software updates within the automotive industry. It  addresses the growing need for a reliable solution that can ensure the safety and integrity of software updates for vehicles.
```

File: blog/2023-12-01-24pullRequests/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## 🎁 24 Pull Requests:
+
+The [24 Pull Requests](https://24pullrequests.com/) event is an annual community-organized celebration, inviting developers to contribute to open source projects during the holiday season. Join us in spreading the joy of collaboration by submitting 24 pull requests between December 1 and December 24.
```
Comment (jhdalek55) [inline blog/2023-12-01-24pullRequests/index.md:16]:
```suggestion
The [24 Pull Requests](https://24pullrequests.com/) event is an annual community-organized celebration in which developers are invited to contribute to open source projects over the holiday season. Join us in spreading the joy of collaboration by submitting 24 pull requests between December 1 and December 24.
```

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
A few small syntactic changes...

File: blog/2023-12-01-24pullRequests/bannerImage.png
Comment (hexsecs) [inline blog/2023-12-01-24pullRequests/bannerImage.png:1]: Can you use the updated Uptane logo? I would like to deprecate the using the old one for new media content.

**Review by hexsecs — CHANGES_REQUESTED** [review]
Just the logo comment everything else LGTM.

**Pull request merged**